addons/web_tour/static/src/js/running_tour_action_helper.js

```diff
@@ -148,12 +148,43 @@ var RunningTourActionHelper = core.Class.extend({
             $to = get_jquery_element_from_selector(to);
         }
 
-        let toCenter = calculateCenter();
+        let toCenter = this._calculateCenter($to, to);
         values.$element.trigger($.Event("mousemove", {which: 1, pageX: toCenter.left, pageY: toCenter.top}));
         // Recalculate the center as the mousemove might have made the element bigger.
-        toCenter = calculateCenter();
+        toCenter = this._calculateCenter($to, to);
         values.$element.trigger($.Event("mouseup", {which: 1, pageX: toCenter.left, pageY: toCenter.top}));
-     },
+    },
+    _drag_move_and_drop: function (values, to) {
+        let initialMoveOffset = [1, 0];
+        let fromSelector;
```

Comment:
This can be removed if https://github.com/odoo/odoo/pull/85778/files#r851259444 is correct and directly defined when assigned later.
(note that you should use const for array as you don't reassign the variable but just change its items)

Reply:
I do not need the default offsets either... they were used by the classic `drag_and_drop`.